cwf/cloud/go/services/cwf/obsidian/models/swagger.v1.yml

```diff
@@ -905,10 +907,30 @@ definitions:
     type: array
     items:
       $ref: '#/definitions/allowed_gre_peer'
-  li_imsis:
-    type: array
-    items:
-      type: string
+  li_ues:
+    type: object
+    required:
+      - imsis
+      - msisdns
+      - macs
+      - ips
```

Comment:
Is there a reason for making these arrays all required?

Reply:
Yeah just easier to explain to partner that this is the format they need to format it in